README.md

```diff
@@ -148,22 +148,22 @@ pre-compiled version of Restor.
 #### Requirements
 
 * Xcode 9+ installed
-* [cocoapods](https://cocoapods.org) installed
+* [bazel](https://bazel.build) installed
 * A valid "Mac Developer" Signing Certificate from Apple
-* Xcode command line tools installed
+* Xcode 9+ command line tools installed
 
 #### Build steps
 
 1. `git clone https://github.com/google/restor.git`
 1. `cd restor`
-1. `pod install`
-1. Find your Team Identifer. Manually selecting the correct Team Identifier might be required if you have multiple developer certificates.
+1. Find your certificate's common name (CN).
     ```bash
-    security find-certificate -p -c "Mac Developer" | openssl x509 -inform pem -subject | perl -ne '/OU=(\w+)\// && print $1'
+    security find-certificate -p -c "Mac Developer" | openssl x509 -inform pem -subject | perl -ne '/CN=([\w :\(\)]+)\// && print $1'
     ```
+1. Edit the `.bazelrc` file, filling in your CN.
```

Comment:
Doesn't bazel make both of these steps optional? If you don't specific the signing cert flag and there's just 1 Dev cert doesn't it just use that by default?

Reply:
>Doesn't bazel make both of these steps optional?

Yes, for non SMJobBless projects.

>If you don't specific the signing cert flag and there's just 1 Dev cert doesn't it just use that by default?

For this project, I intentionally make the lookup fail. Forcing you to fill in the common name. 
https://github.com/google/restor/blob/af1ede907d547c0c7b004f9c533b72c936c6b299/.bazelrc#L1

`ctx.fragments.objc.signing_certificate_name` does not do any resolving against the keychain. So for SMJobBless to work, you need to set the complete common name. Or we would need to do the resolving in Starlark :/
https://github.com/google/restor/blob/af1ede907d547c0c7b004f9c533b72c936c6b299/signing_certificate.bzl#L8